docs/src/getting_started.md

```diff
@@ -39,6 +46,24 @@ pip3 install stata_kernel
 python3 -m stata_kernel.install
 ```
 
+### Jupyter
+
+If you chose to install Anaconda you already have Jupyter and Jupyter Lab installed.
+
+If you downloaded Miniconda, you need to install Jupyter and optionally Jupyter Lab:
```

Comment:
I'd say Jupyter **or** Jupyter Lab, there's no reason both are needed. And I'd suggest a slight nudge towards Jupyter lab

Reply:
Also you can probably suggest installing `notebook`, not `jupyter`.

Reply:
Thanks, agreed.